content/docs/getting-started/hello-world.md

```diff
@@ -33,9 +33,9 @@ channels:
           type: string
           pattern: '^hello .+$'</code></pre>
 
-The first line tells us this is an AsyncAPI document and it's using version 2.0.0 of the specification. This line doesn't have to be the first one but it's a recommended practice.
+The first line of the specification starts with the document type (AsyncAPI) and the version (2.0.0) as a recommended practice.
```

Comment:
@lifewingmate I meant the recommended practice is to put this line as the first one. However, the line is mandatory. It just doesn't have to be the first.